magmi/plugins/5b5/general/attributesetimport/multi_dim_array.php

```diff
@@ -75,6 +75,7 @@ public function offsetGet($name)
      *
      * @see ArrayIterator::offsetSet()
      */
+#[\ReturnTypeWillChange]
```

Comment:
why is this needed? what kind of error does it fix?

Reply:
sorry can not remember think i when i was working out how to fix i was putting in comment to help me
as i was saying the to my self this may change to fix the error